### PR TITLE
Profile and Contacts

### PR DESCRIPTION
**Pull request by Disquietude**
Connecting the front-end and back-end to be able to edit user profiles and add contacts.

### CODE REVIEW

**Review by Harukine — APPROVED** [review]
Looks great

File: client/src/components/Home/Sidebar/Profile/ProfileModal.js [hunk 1]
```diff
@@ -5,19 +5,36 @@ import TextField from "@material-ui/core/TextField";
 import { withStyles } from "@material-ui/styles";
 
 import styles from '../../../../styles/ModalStyles';
+import StyledButton from '../../../../styles/StyledButton';
 
 class ProfileModal extends Component {
   state = { 
-    firstName: '',
-    lastName: ''
+    firstName: this.props.profile.firstName,
```
Comment (aeciorc) [inline client/src/components/Home/Sidebar/Profile/ProfileModal.js:12]: Any reason why you do this instead of just using the props?

**Review by aeciorc — APPROVED** [review]
Looks good, one comment below

File: client/src/components/Home/Sidebar/Profile/ProfileModal.js (hunk 1 above)
Comment (Disquietude) [reply client/src/components/Home/Sidebar/Profile/ProfileModal.js:12]: The TextInputs are controlled components so their values are stored in the component state, but their initial values come from the props.

Comment (aeciorc) [reply client/src/components/Home/Sidebar/Profile/ProfileModal.js:12]: Got it, makes sense

**Pull request merged**